src/nunit.analyzers/Constants/ContainsConstraintWrongActualTypeConstants.cs

```diff
@@ -0,0 +1,9 @@
+namespace NUnit.Analyzers.Constants
+{
+    internal static class ContainsConstraintWrongActualTypeConstants
+    {
+        public const string Title = "Wrong actual type used with ContainsConstraint.";
+        public const string Message = "The ContainsConstraint cannot be used with '{0}' actual argument.";
+        public const string Description = "The ContainsConstraint constraint requires actual argument to be either string, or collection of strings.";
```

Comment:
```suggestion
        public const string Title = "Wrong actual type used with ContainsConstraint.";
        public const string Message = "The ContainsConstraint cannot be used with an actual value of type '{0}'.";
        public const string Description = "The Contains constraint requires actual value to be either a string or a collection of strings.";
```

Reply:
You suggest to leave it as `ContainsConstraint` in title and message, but change to `Contains constraint` in description. Why not consistent?

Reply:
I think I wanted to changed the wording as the "The ContainsConstraint constraint" sounded a bit strange (we don't have other places where we have the substring "...Constraint constraint..."), and I found some places where we write "SameAs constraint" or "String Constraint". But you are right that this is inconsistent.

Perhaps the better option would be (also more consistent with the other analyzers):
```
        public const string Description = "The ContainsConstraint requires actual value to be either a string or a collection of strings.";
```

Reply:
Yep, already done

Reply:
Great. I had only time to look at the comment, so I did not see the code changes.